modules/lib/lib-content/src/main/resources/lib/xp/content.ts

```diff
@@ -48,6 +60,50 @@ export type {
     Content,
     Component,
     Region,
+    QueryDsl,
```

Comment:
This is a generic comment for all Enonic code. Why aren't we sorting alphabetically? It becomes hard to see what's there or not when it's "random" order.

Reply:
I recommend looking into having the tooling (like eslint) doing the job if this is something you are going for. 

_(Disclamer: I have not checked if eslint actually can do this.)_

Reply:
The formatting is really I hard thing to discuss. The only working solution that I can see right now is Prettier.

But let me tell you about the whole picture first. It so happened that the core team uses the IntelliJ IDEA and the shared config where various code styles are defined. In the past we tried different approaches, even git precommit hooks + lint-staged + prettier. These practices have not caught on though. It's kind off hard to keep the current styles, make the formatting process simple, not demanding and non-blocking.

I think that we'll discuss the styling improvements this month and, probably expand our eslint config with prettier in the future, but definitely not right now and not in this PR.